modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -440,12 +440,13 @@ export async function putListTypeEntry({
 	});
 }
 
-export async function updateRelationship({
+export async function putObjectRelationship({
 	objectRelationshipId,
 	...others
 }: ObjectRelationship) {
 	return await save({
 		item: others,
+		method: 'PUT',
```

Comment:
This is not needed because the default value is "PUT", but we can keep it for the sake of code legibility.

Reply:
I missed this, thanks @naraandrade

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/2571/commits/5696b205298527682c82776779d9cbaac9c50b91)